src/pfs/pfs.proto

```diff
@@ -258,10 +261,18 @@ message SquashCommitSetRequest {
   CommitSet commit_set = 1;
 }
 
+message SquashCommitSetsRequest {
```

Comment:
Is this primarily intended for the case where there are dependent commit sets that need to be squashed first? If so, we may be better off with just having a flag for squash which will handle that automatically. So, a user could squash, get back an error that explains the dependency, then use the flag to recursively squash commit sets until the original commit set can be squashed.